tlspuffin/src/tls/key_schedule.rs

```diff
@@ -18,22 +19,18 @@ pub fn tls13_handshake_traffic_secret(
     psk: &Option<Vec<u8>>,
     client: bool,
     group: &NamedGroup,
-) -> Result<(&'static SupportedCipherSuite, Prk, KeyScheduleHandshake), FnError> {
-    let client_random = &[1u8; 32]; // todo see op_random() https://github.com/tlspuffin/tlspuffin/issues/129
-    let suite = &crate::tls::rustls::tls13::TLS13_AES_128_GCM_SHA256; // todo see op_cipher_suites() https://github.com/tlspuffin/tlspuffin/issues/129
-    let key_schedule = dhe_key_schedule(suite, group, server_key_share, psk)?;
+    client_random: &Random,
+    suite: &SupportedCipherSuite,
+) -> Result<(Prk, KeyScheduleHandshake), FnError> {
+    let key_schedule = dhe_key_schedule(&suite, group, server_key_share, psk)?;
```

Comment:
Only option?: `dhe_key_schedule`

Reply:
`dhe_key_schedule` seems to be the only options, this seems to derive the secrets according to the TLS spec : this function as multiple code paths depending on the presence of a PSK and keyshare and call the HKDF associated with the cipher suite, I don't think there are other possibilities here.